### PR TITLE
Display changelogs for WordPress plugins

### LINKED ISSUE #6: Display changelogs for WordPress plugins
It would be nice if `composer update` or `composer outdated` would show a link to the changelog of a plugin.

Example: https://packagist.org/packages/pyrech/composer-changelogs

### PR DESCRIPTION
**Pull request by swissspidy**
Fixes #6.

### CODE REVIEW

**Comment by ocean90** [comment]
What are the steps to test this PR?

demo

**Comment by swissspidy** [comment]
Besides the unit tests you could install the branch globally and run `composer update` on a project to see the output.

File: src/PluginChangelogPlugin.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+	public function activate( Composer $composer, IOInterface $io ) {
+		$this->composer = $composer;
+		$this->io       = $io;
+		$this->helper   = new WordPressPluginHelper();
+	}
+
+	public static function getSubscribedEvents() {
+		return [
+			PackageEvents::PRE_PACKAGE_UPDATE => [
```
Comment (ocean90) [inline src/PluginChangelogPlugin.php]: I think `PackageEvents::POST_PACKAGE_UPDATE` is better for this.

File: src/PluginChangelogPlugin.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	 * abandonment by its developer, or even security issues.
+	 *
+	 * @param PackageEvent $event The current event.
+	 */
+	public function printLinkToChangelog( PackageEvent $event ) {
+		$package = $event->getOperation()->getTargetPackage();
+
+		if ( $this->helper->isWordPressPlugin( $package ) ) {
+			$this->io->write( sprintf(
+				'Changelog: %s',
```
Comment (ocean90) [inline src/PluginChangelogPlugin.php]: Let's add 4 spaces before "Changelog".

After:
after

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Review by ocean90 — APPROVED** [review]

**Pull request merged**